server/test/unit/server/db/test_connection.py

```diff
@@ -72,6 +77,34 @@ def test_seeds_from_config(self, mock_connect_seeds, mock_mongoengine):
                                                                             'secondhost:5678'],
                                               database)
 
+    @patch('pulp.server.db.connection.mongoengine')
+    @patch('pulp.server.db.connection._connect_to_one_of_seeds')
+    def test_multiple_seeds_no_replica_set(self, mock_connect_seeds, mock_mongoengine):
+        mock_mongoengine.connect.return_value.server_info.return_value = {'version': '2.6.0'}
+        mock_connect_seeds.return_value.server_info.return_value = {'version': '2.6.0'}
+        seeds = "firsthost:1234,secondhost:5678"
+        config.config.set('database', 'write_concern', 'majority')
+        config.config.set('database', 'seeds', seeds)
+        try:
+            connection.initialize()
+        except PulpCodedException as connection_error:
+            self.assertEqual(connection_error.error_code.message, error_codes.PLP0041.message)
+
+    @patch('pulp.server.db.connection.mongoengine')
+    @patch('pulp.server.db.connection._connect_to_one_of_seeds')
+    def test_invalid_write_concern(self, mock_connect_seeds, mock_mongoengine):
+        mock_mongoengine.connect.return_value.server_info.return_value = {'version': '2.6.0'}
+        mock_connect_seeds.return_value.server_info.return_value = {'version': '2.6.0'}
+        seeds = "firsthost:1234,secondhost:5678"
+        replica_set = 'fakeReplica'
+        config.config.set('database', 'write_concern', 'blah')
+        config.config.set('database', 'seeds', seeds)
+        config.config.set('database', 'replica_set', replica_set)
+        try:
+            connection.initialize()
+        except PulpCodedException as connection_error:
+            self.assertEqual(connection_error.error_code.message, error_codes.PLP0043.message)
```

Comment:
@mhrivnak I was using self.assertRaises to check for PulpCodedException. I wasn't able to check that it was a specific one actually. So I changed the test to this format so I could be more specific. Unless you can suggest how I can use self.assertRaises with a particular PulpCodedException,  I will add a failure statement after line 104.